### PR TITLE
Update eks helm guide for AWS PCA

### PR DESCRIPTION
**Pull request by pschisa**
updating cert manger instructions when using AWS PCA plugin

### CODE REVIEW

File: docs/pages/deploy-a-cluster/helm-deployments/aws.mdx [hunk 1]
```diff
@@ -82,6 +82,13 @@ these options:
 You can use `cert-manager` to provision and automatically renew TLS credentials
 by completing ACME challenges via Let's Encrypt.
 
+`cert-manager` can also be used with AWS Private CAs in EKS using the `aws-privateca-issuer`
+plugin.  If using an AWS PCA, you will need to ensure you set `addCommonName: true` in your
+values file.  You will also need to get the certificate authority certificate for the CA
+(`aws acm-pca get-certificate-authority-certificate --certificate-authority-arn <arn>`),
+upload the full certificate chain to a secret, and reference the secret with 
+`tls.existingCASecretName` in the values file.
```
Comment (hugoShaka) [inline docs/pages/deploy-a-cluster/helm-deployments/aws.mdx:90]: Can you link to the reference docs at least for `tls.existingCASecretName`? It contains instructions on how to create the secret with the correct format.

**Review by hugoShaka — APPROVED** [review]

**Review by tigrato — APPROVED** [review]

File: docs/pages/deploy-a-cluster/helm-deployments/aws.mdx [hunk 2]
```diff
@@ -82,6 +82,15 @@ these options:
 You can use `cert-manager` to provision and automatically renew TLS credentials
 by completing ACME challenges via Let's Encrypt.
 
+`cert-manager` can also be used with AWS Private CAs in EKS using the
```
Comment (stevenGravy) [inline docs/pages/deploy-a-cluster/helm-deployments/aws.mdx:85]:
```suggestion
`cert-manager` can also be used with AWS Private CAs (PCA) in EKS using the
```

File: docs/pages/deploy-a-cluster/helm-deployments/aws.mdx [hunk 3]
```diff
@@ -82,6 +82,15 @@ these options:
...
 
+`cert-manager` can also be used with AWS Private CAs (PCA) in EKS using the
+`aws-privateca-issuer` plugin.  If using an AWS PCA, you will need to 
+[ensure you set](../../reference/helm-reference/teleport-cluster.mdx#highavailabilitycertmanageraddcommonname)
+`highAvailability.certManager.addCommonName: true` in your values file.  You will also need to get the certificate authority
+certificate for the CA (`aws acm-pca get-certificate-authority-certificate --certificate-authority-arn <arn>`), 
+upload the full certificate chain to a secret, and 
+[reference the secret](../../reference/helm-reference/teleport-cluster.mdx#tlsexistingcasecretname)
+with `tls.existingCASecretName` in the values file.
+
```
[the remark this answers is not in the document]
Comment (pschisa) [reply docs/pages/deploy-a-cluster/helm-deployments/aws.mdx]: @ptgott I have moved the actual instructions for the values file to the values tabs for cert manager as a note. LMK what you think.

[the remark this answers is not in the document]
Comment (ptgott) [reply docs/pages/deploy-a-cluster/helm-deployments/aws.mdx]: Thanks! Before I take a look, should there also be changes to the `Issuer` instructions? It seems counterintuitive to me that you would need to install a new issuer plugin, but the `Issuer` you would create would still use Let's Encrypt.

[the remark this answers is not in the document]
Comment (ptgott) [reply docs/pages/deploy-a-cluster/helm-deployments/aws.mdx]: I'm going to bring @webvictim into this conversation as well. Gus, if you think the new instructions are complete and followable, I'm happy to give this one a ✅ . Thanks!

[the remark this answers is not in the document]
Comment (pschisa) [reply docs/pages/deploy-a-cluster/helm-deployments/aws.mdx]: From my understanding, when a user deploys the private CA integration with cert-manger, the plugin will configure the issuer (https://github.com/cert-manager/aws-privateca-issuer)

I think then that the section `You'll need to replace these values in the Issuer example below:` covers the required changes a user would need to make to integrate with their already deployed issuer if utilizing AWS PCA.

**Review by ptgott — APPROVED** [review]

File: docs/pages/deploy-a-cluster/helm-deployments/aws.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (ptgott) [reply docs/pages/deploy-a-cluster/helm-deployments/aws.mdx]: Approved on the assumption that a user will know where to add the appropriate info!

**Pull request merged**